### DISCUSSION TITLE
Does skywalking oap have a metadata information interface?

### DISCUSSION THREAD

**Discussion by mark8s** [apache/skywalking#11610] [Q&A] [answered]
What I mean is, does skywalking oap have its own metadata information interface. For example, I now have an oap address of skywalking: 10.31.0.111:11800. I want to obtain its version, environment variables, and even health status through its API...

Is there such an API?

**Comment by wu-sheng** [comment, marked as answer]
- Healthy Check APIs

https://skywalking.apache.org/docs/main/v9.6.0/en/setup/backend/backend-health-check/

- Version, also GraphQL

```graphql
{"query":"query version { version }","variables":{}}
```

- Environment Variables for OAP. This doesn't exist, but would have a new API since 9.7.0(releasing...). 

https://skywalking.apache.org/docs/main/next/en/debugging/config_dump/

**Reply by mark8s** [reply]
@wu-sheng ok，tks~
